test/arrays/return/returnArbitraryBadEltType.good

```diff
@@ -0,0 +1,2 @@
+returnArbitraryBadEltType.chpl:1: In function 'inc':
+returnArbitraryBadEltType.chpl:2: error: type mismatch in assignment from real(64) to int(64)
```

Comment:
Since you're making up the error message at this point I think it'd be nice to say something more like `array element type mismatch in return` (and you can still put the types if you want).